lib/src/flags.dart

```diff
@@ -0,0 +1,607 @@
+// GENERATED FILE, timestamp: 2023-09-30 15:11:37.097628
+
+import 'package:circle_flags/circle_flags.dart';
+
+/// List of available flags(iso codes) for [CircleFlag].
+abstract class Flags {
+  static const values = <String>[
+    'ac',
```

Comment:
I find it cleaner if the fields could be used here instead of the strings.

Reply:
done